apps/macos/src/index.tsx

```diff
@@ -1,17 +1,8 @@
 'use strict';
 
 import {FluentTesterApp} from '@fluentui-react-native/tester';
-import * as React from 'react';
-import {AppRegistry, SafeAreaView} from 'react-native-macos';
```

Comment:
Fun note: Big Sur just added safe areas to macOS as a better means of layout relative to a title bar than the current solution of the layout is either full screen or cannot underlap the title bar. Still the right change here, but we may need to re-evaluate as we look at this with the 11.0 SDK

Reply:
I remember that! I vaugely remember Safe Area's being a new thing. I basically had two thoughts:

1) Leave it in as the extra JSX doesn't do anything right now and I'm sure SafeAreaView isn't implemented for react-native-macOS yet. This is the futureproof approach

2) Remove it and add it when it becomes a necessity.

As you can see, I've gone between both :D